dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESIndexAPI.java

```diff
@@ -815,14 +827,18 @@ public void openIndex(String indexName) {
     }
 
     public List<String> getClosedIndexes() {
-        Client client=new ESClient().getClient();
-        ImmutableOpenMap<String,IndexMetaData> indexState=client.admin().cluster().prepareState().execute().actionGet()
-                                                           .getState().getMetaData().indices();
-        List<String> closeIdx=new ArrayList<String>();
-        for ( ObjectCursor<String> idx : indexState.keys() ) {
-            IndexMetaData idxM = indexState.get( idx.value );
-            if ( idxM.getState().equals( State.CLOSE ) ) {
-                closeIdx.add( idx.value );
+        Client client = new ESClient().getClient();
+        ImmutableOpenMap<String, IndexMetaData>
+            indexState =
+            client.admin().cluster().prepareState().execute().actionGet()
+                .getState().getMetaData().indices();
+
+        List<String> closeIdx = new ArrayList<>();
```

Comment:
I think you can do something like

index.entrySet().stream().filter(entryIdxM -> State.CLOSE.equals(entryIdxM.getValue().getState()).map(entry -> entry.getKey().value).collect(CollectionUtils.toImmutableList());

Reply:
@jdotcms what about? 
`index.values().stream().filter(index -> State.CLOSE.equals(index.getState()).collect(CollectionUtils.toImmutableList());`

Reply:
if you mean **indexState**.entrySet().stream() or **indexState**.values().stream(), none of those lines compile